test/xpu/quantization/core/test_quantized_tensor_xpu.py

```diff
@@ -0,0 +1,184 @@
+# Owner(s): ["module: intel"]
```

Comment:
This file is copied from uts in pytorch?

Reply:
Can we reuse original uts?

Reply:
Yes, the code is copied from pytorch. Some UT uses hardcoded `cuda` device but are suitable for Intel GPU, so `cuda` is replaced with `xpu` for each ocurrence.